backend/inspirehep/records/api/literature.py

```diff
@@ -305,6 +305,7 @@ def add_file(
         new_key = hash_data(file_data)
         mimetype = magic.from_buffer(file_data, mime=True)
         size = len(file_data)
+        file_data = BytesIO(file_data)
```

Comment:
what's the point of this?

Reply:
Here yes. Main part is disabling threads on `upload_fileobj`.